charts/seed-operatingsystemconfig/original/templates/osc.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
 spec:
   type: {{ required ".osc.type is required" .Values.osc.type }}
   purpose: {{ required ".osc.purpose is required" .Values.osc.purpose }}
+  providerConfig: 
+{{ .Values.osc.providerConfig | indent 2 }}
```

Comment:
```suggestion
{{ .Values.osc.providerConfig | indent 4 }}
```

Reply:
By the way, this statement throws an exception when running `make verify`
```
Error: render error in "original/templates/osc.yaml": template: original/templates/osc.yaml:13:39: executing "original/templates/osc.yaml" at <4>: invalid value; expected string
```

Reply:
You probably have to do:

```yaml
  {{- if .Values.osc.providerConfig }}
  providerConfig: 
{{ .Values.osc.providerConfig | indent 4 }}
  {{- end }}
```